ui/src/app/store/machine/types/base.ts

```diff
@@ -92,13 +92,14 @@ export type MachineDetails = BaseMachine &
     installation_status: number;
     interface_test_status: TestStatus;
     interfaces: NetworkInterface[];
-    last_sync: string;
+    is_sync_healthy?: boolean;
+    last_sync?: string;
     license_key: string;
     memory_test_status: TestStatus;
     metadata: NodeMetadata;
     min_hwe_kernel: string;
     network_test_status: TestStatus;
-    next_sync: string;
+    next_sync?: string;
```

Comment:
Made the machine sync details properties optional. I tried a conditional type which worked nicely (and reflected what types we can actually get at runtime), but that didn't work with `cooky-cutter` (seems like due to some of its internal type merge logic). Would you have any ideas how to make it work with `cooky-cutter`?

```
type MachineHwSyncDetails =
  | {
      enable_hw_sync: true;
      last_sync: string;
      next_sync: string;
      is_sync_healthy: boolean;
      sync_interval: Seconds;
    }
  | {
      enable_hw_sync: false;
    };

export type MachineDetails = BaseMachine &
  TimestampFields &
  MachineSyncDetails & {
...
```